api/v1/nifidataflow_types.go

```diff
@@ -1,6 +1,7 @@
 package v1
 
 import (
+	"github.com/konpyutaika/nifikop/api/v1alpha1"
```

Comment:
```suggestion
	v1 "github.com/konpyutaika/nifikop/api/v1"
```
as it used by NifiDataflow resource that is in `v1`, you should use the `v1` not the `v1alpha1`.